Makefile.COMMON

```diff
@@ -37,15 +37,24 @@
 # The including Makefile may define any number of extra targets that are
 # specific to that project.
 
-VERSION ?= $(error VERSION not set in including Makefile)
```

Comment:
As described in the comment at the top of this file, Makefile.COMMON is shared between many repositories, please don't make any changes here.